### DISCUSSION TITLE
I cannot use Copilot, even I approved as a student.

### DISCUSSION THREAD

**Discussion by Nurxat21** [community/community#114347] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/75790366/e4200e91-a888-4cca-9e6e-f6a17eca008c)
![image](https://github.com/community/community/assets/75790366/9648dc8e-86e9-4838-ac1b-4cb3674330b0)
![image](https://github.com/community/community/assets/75790366/a53b326a-ff52-402b-a219-08a687b35477)
So how could I use copilot? I have tried many solutions that provided by the Github. But still not worked. What should I do?

**Comment by davecheney** [comment, marked as answer]
I am sorry, we cannot assist you with this issue via this forum. Please contact support https://support.github.com/contact
